app/views/marine/index.html.erb

```diff
@@ -32,26 +32,35 @@
 
 
 <%= render partial: "marine/sections/coverage", locals: {
-  cms_page: @cms_page,
-  
+  cms_page: @cms_page  
 } %>
 
-
-
-<%= render partial: "marine/sections/national" %>
+<%= 
+  render partial: "marine/sections/national" 
+%>
 
 <section class="container--medium">
   <h2><%= cms_fragment_render(:section_size_title, @cms_page) %></h2>
   <div><%= cms_fragment_render(:section_size_content, @cms_page) %></div>
 </section>
 
-<%= render partial: "marine/sections/size_growth" %>
+<%= render partial: "marine/sections/size_growth", locals: {
+  regionsTopCountries: @regionsTopCountries,
+} %>
 
-<%= render partial: "marine/sections/coverage_growth" %>
+<%= render partial: "marine/sections/coverage_growth", locals: {
+  regionCoverage: @regionCoverage,
+} %>
 
 <%= render partial: "marine/sections/coverage_map" %>
 
-<%#= render partial: "partials/cards/sites", locals: { cards: @wdpa } %>
+<section class="container">
+  <%= render partial: "partials/cards/sites", locals: { 
+    cards: cards, 
+    title: "#{t('global.area-types.wdpa')} (#{wdpaMarineTotal})",
+    url: urlViewAll
```

Comment:
@stacytalbot  These 3 variables (cards, wdpaMarineTotal and urlViewAll) are all undefined so the page won't load.
I wonder if it's related to an attempt in fixing merge conflicts?